.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+name: Build artifacts
+
+# ==== NOTE: do not rename this yml file or the run_number will be reset ====
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use .NET Core 3.1 SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '3.1.x'
+          source-url: https://nuget.pkg.github.com/graphql-dotnet/index.json
+        env:
+          NUGET_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      - name: Install dependencies
+        run: dotnet restore
+        env:
+          DOTNET_NOLOGO: true
+          DOTNET_CLI_TELEMETRY_OPTOUT: true
+      - name: Build solution (release config)
+        run: dotnet build --no-restore -c Release -p:NoWarn=CS1591 -p:VersionSuffix=$GITHUB_RUN_NUMBER
```

Comment:
```suggestion
        run: dotnet build --no-restore -c Release -p:NoWarn=CS1591 -p:VersionSuffix=$GITHUB_RUN_NUMBER -p:ContinuousIntegrationBuild=true
```

Reply:
Already covered unless you think we should move it from build props to here

Reply:
:) Actually, I first wanted to set this property in the project file too as you did. I just did not find necessary docs about `GITHUB_ACTION` and other props so I did the same via `-p:` option. I think that in case I will redo my projects to fit your way.